common/src/test/java/io/pravega/common/security/JwtUtilsTest.java

```diff
@@ -29,8 +32,9 @@ public void testExtractExpirationTimeReturnsNullIfExpInBodyIsNotSet() {
         //        "aud": "segmentstore",
         //        "iat": 1516239022
         //     }
+        JwtBody jwtBody = JwtBody.builder().subject("1234567890").audience("segmentstore").issuedAtTime(1516239022L).build();
         String token = String.format("%s.%s.%s", "base64-encoded-header",
-                JwtBody.builder().subject("1234567890").audience("segmentstore").issuedAtTime(1516239022L).build(),
+                Base64.getEncoder().encodeToString(jwtBody.toString().getBytes(StandardCharsets.US_ASCII)),
```

Comment:
StandardCharsets.UTF8

Reply:
Since ASCII is a subset of UTF-8, and we use only ASCII characters in this particular test, I think it should be OK to use the specified charset.